resources/views/Listings/listing_edit.blade.php

```diff
@@ -0,0 +1,112 @@
+@extends('layout', [
+    'title' => 'wijzig product'
+])
+
+@section("content")
+    @if (session('success'))
+        <div class="alert alert-success w-25">
+            <li>
+                {{ session('success') }}
+            </li>
+        </div>
+    @endif
+
+    @if (session('errors'))
+        <div class="alert alert-success w-25">
+            <li>
+                {{ session('success') }}
+            </li>
+        </div>
+    @endif
+    <div class="text-center">
+        <a href="{{route("advert.listings")}}" style="text-decoration: none" class="text-uppercase">Terug</a>
```

Comment:
deze knop op een beter plek. Waarschijnlijk onder aan de rechterkant. of rechtsbovenin maar duidelijker